packages/vuetify/src/labs/VHotkey/VHotkey.scss

```diff
@@ -0,0 +1,106 @@
+@use '../../styles/tools';
+@use './variables' as *;
+
+
+@include tools.layer('components') {
+ .v-hotkey {
+    align-items: center;
+    display: inline-flex;
+    gap: $hotkey-gap;
+    vertical-align: middle;
+    line-height: $hotkey-line-height;
+
+    &__key-icon {
+      .v-icon {
+        max-width: $hotkey-icon-max-width;
+        min-width: unset;
+      }
+    }
+
+    &__divider {
+      align-items: center;
+      display: inline-flex;
+      opacity: $hotkey-divider-opacity;
+      font-size: $hotkey-divider-font-size;
+    }
+
+    &__combination {
+      align-self: baseline;
+      display: flex;
+      gap: $hotkey-combination-gap;
+    }
+
+    &__key.v-kbd {
+      min-height: unset;
+      font-size: $hotkey-font-size;
+      line-height: $hotkey-line-height;
+      padding: $hotkey-padding;
+      min-width: $hotkey-min-width;
+    }
+
+    &__key-symbol.v-kbd {
+      line-height: normal;
+      font-size: 1em;
+    }
+
+    &--inline {
+      align-items: baseline;
+      max-height: 1lh;
+      vertical-align: baseline;
+      font-size: $hotkey-inline-font-size;
```

Comment:
dividers within `--inline` ends up with 10.5px font size. Should they be set to 1em as well? Perhaps with a dedicated Sass variable?